docs/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  overflow-x: hidden;
+}
```

Comment:
Having to hide `overflow-x` on body is a sign that something's off in your styling. Fixing the cause of the overflow and fixing it would be better than painting over the problem.

Reply:
This one is resolved once I added responsiveness to the site. I will send updated PR with this one removed.